pkg/log/log.go

```diff
@@ -92,7 +92,7 @@ func newConfig(level zapcore.Level, encoding EncodingType) zap.Config {
 func newEncoderConfig(encoding EncodingType) zapcore.EncoderConfig {
 	if encoding == HumanizeEncoding {
 		return zapcore.EncoderConfig{
-			TimeKey:        "eventTime",
+			TimeKey:        "",
```

Comment:
After zap 1.14.1, we need to set `EncodeTime` if we set `TimeKey`. So the test for piped's logger is failed.
https://github.com/uber-go/zap/pull/791

But currently, piped's logger doesn't show a timestamp when the log encoding is `HumanizeEncoding`.
So I fixed not to set `TimeKey`.

Reply:
[question] How does this affect other log encoding settings (which are not humanized)? Afaik, some teams are using piped logs with JSON encoding now, so we need to ensure nothing is broken.

Reply:
@khanhtc1202 
I think it doesn't affect other log formats because `EncodeTime` is set for the other format. 
https://github.com/pipe-cd/pipecd/blob/9ddcc8e9f80d441a05b6873a3719df5c79cca002/pkg/log/log.go#L92C1-L121C2

The problem is that when not set `EncodeTime ` despite `TimeKey` is set.
https://github.com/uber-go/zap/pull/791

I also checked the log for each formats below and `eventTime` is inserted when using json and console log format.

json format
```
% CONFIG_FILE=./.dev/local2/piped.yaml LOG_ENCODING=json make run/piped                                                                                                                      (git)-[add-k8s-e2e-test]
go run cmd/piped/main.go piped --tools-dir=/tmp/piped-bin --config-file=./.dev/local2/piped.yaml --insecure=false --log-encoding=json
{"severity":"INFO","eventTime":1734398586.566765,"logger":"piped.piped","caller":"piped/piped.go:182","message":"successfully configured ssh-config","serviceContext":{"service":"piped.piped","version":"unspecified"}}
{"severity":"INFO","eventTime":1734398586.568584,"logger":"piped.piped.tool-registry","caller":"toolregistry/registry.go:57","message":"successfully loaded the pre-installed tools","serviceContext":{"service":"piped.piped","version":"unspecified"},"tools":{"helm":{},"kubectl-1.18.5":{},"kubectl-1.30.0":{},"kubectl-1.30.5":{},"kustomize":{}}}
{"severity":"INFO","eventTime":1734398589.319672,"logger":"piped.piped.notifier","caller":"notifier/notifier.go:116","message":"all 0 notifiers have been started","serviceContext":{"service":"piped.piped","version":"unspecified"}}
...
```

console format
```
% CONFIG_FILE=./.dev/local2/piped.yaml LOG_ENCODING=console make run/piped                                                                                                                   (git)-[add-k8s-e2e-test]
go run cmd/piped/main.go piped --tools-dir=/tmp/piped-bin --config-file=./.dev/local2/piped.yaml --insecure=false --log-encoding=console
1.734398656024647e+09	INFO	piped.piped	piped/piped.go:182	successfully configured ssh-config	{"serviceContext": {"service": "piped.piped", "version": "unspecified"}}
1.734398656024822e+09	INFO	piped.piped.tool-registry	toolregistry/registry.go:57	successfully loaded the pre-installed tools	{"serviceContext": {"service": "piped.piped", "version": "unspecified"}, "tools": {"helm":{},"kubectl-1.18.5":{},"kubectl-1.30.0":{},"kubectl-1.30.5":{},"kustomize":{}}}
1.734398657369169e+09	INFO	piped.piped.notifier	notifier/notifier.go:116	all 0 notifiers have been started	{"serviceContext": {"service": "piped.piped", "version": "unspecified"}}
...
```

humanize format
```
% CONFIG_FILE=./.dev/local2/piped.yaml LOG_ENCODING=humanize make run/piped                                                                                                                  (git)-[add-k8s-e2e-test]
go run cmd/piped/main.go piped --tools-dir=/tmp/piped-bin --config-file=./.dev/local2/piped.yaml --insecure=false --log-encoding=humanize
successfully configured ssh-config
successfully loaded the pre-installed tools	{"tools": {"helm":{},"kubectl-1.18.5":{},"kubectl-1.30.0":{},"kubectl-1.30.5":{},"kustomize":{},"kustomize-5.0.1":{},"kustomize-5.3.0":{}}}
all 0 notifiers have been started
```